projects/laji-ui/src/lib/mixins/_base.scss

```diff
@@ -209,4 +209,45 @@
     margin: 20px 0;
     border-top: 1px solid $neutral-4;
   }
-}
\ No newline at end of file
+
+  typeahead-container, .laji-map {
```

Comment:
I would prefer if the base styles would remain decoupled from laji-map and bootstrap. The laji-map and bootstrap related stuff could be moved eg. to `laji.scss` where the base styles are `@include`d.

Reply:
Moved there